### DISCUSSION TITLE
Some designs for the app icon

### DISCUSSION THREAD

**Discussion by eliandoran** [TriliumNext/Notes#203] [Polls]
A friend of mine has volunteered some potential app icons for Trilium Next.

Due to GitHub polls being limited to 8 choices maximum we first have to select a row and then we'll further refine the poll based on which one has the most answers.

1. ![image](https://github.com/user-attachments/assets/bdf7262b-bc8b-41a6-8301-28b666e39460)
2. ![image](https://github.com/user-attachments/assets/e1d7a12c-9795-4cbd-a5ab-6baa7529966d)
3. ![image](https://github.com/user-attachments/assets/a4bce6e6-64e6-4587-ac3c-44f198322874)
4. ![image](https://github.com/user-attachments/assets/fef997fe-a50a-4a26-8462-d768e70ea05a)
5. ![image](https://github.com/user-attachments/assets/0ee19940-008f-4799-8ebd-a095c35f1f7d)

**Comment by root-hal9000** [comment]
I like 3 (especially 3A), but with the color scheming of 5 :)
I say 3 because it's the only one the really distinguishes itself the most from the original though, which would be important since it's confusing if too similar. That said, I would prefer more abstraction/simplicity (the veins in the leaf of no. 3 create too much complexity IMHO)

EDIT: verbally changing my vote towards 5 actually... but we do need more differentiation - perhaps in changing the colors, or making the leaves more abstract as symbols

**Reply by meichthys** [reply]
I also agree with a more simplistic design, similar to number 3, but perhaps more differentiating.

**Comment by maphew** [comment]
I would aim for something more reminiscent to the real world flowers. While a number of varieties have wavy or rippley edges they're not usually spiky, and I don't think I've seen any photos or botanical drawings that sport a strong central vein with branches.

**Comment by zerebos** [comment]
To me row 3 would symbolize that while we are still Trilium, we are a sleeker and smoother evolution of it. Hence 3 has my vote.

**Reply by root-hal9000** [reply]
> To me row 3 would symbolize that while we are still Trilium, we are a sleeker and smoother evolution of it. Hence 3 has my vote.

Yeah, same here - not exactly that as it is now, but I like how you put the symbolism of it. I think a more simplified/minimalist version of it is perfect

**Comment by Alumniminium** [comment]
5 in purple!

**Comment by root-hal9000** [comment]
@eliandoran didn't someone share another one in the Matrix chat? I can't find it right now but I remember it being something I liked (something like a streamlined version of no.3) - maybe we can also share that here for all to see?

**Reply by eliandoran** [reply]
@root-hal9000 ,

It's a design by `jerakin` on Matrix, available here:

![image](https://github.com/user-attachments/assets/86b1d8a0-2cb8-4476-bc9c-adba381687f5)

I could update the poll but apparently that means that all votes would be reset. If people are OK with that, sure.

**Reply by root-hal9000** [reply]
Unless we're going in a completely different direction, this is my favorite. Although I would maybe suggest using different colors to not confuse with the original

**Reply by Alumniminium** [reply]
Not purple, 0/10

**Reply by eliandoran** [reply]
@Alumniminium , I'm fond of the purple tint as well. :)

**Reply by Jerakin** [reply]
Thanks for posting it (I am jerakin here too)


Edit: Quick paint over of it as purple
![image](https://github.com/user-attachments/assets/2bbbe3a5-86d7-4a31-b3b7-0958e06e9e10)

**Reply by Alumniminium** [reply]
![image](https://github.com/user-attachments/assets/fc50c41b-d6a5-494e-a419-cef3105bced0)

**Reply by root-hal9000** [reply]
@Jerakin that's fantastic work.
@eliandoran I see enough support happening for a re-poll haha

**Reply by zerebos** [reply]
I would still prefer row 3 over this personally, and I'm not a fan of the purple logo, it feels both off-brand and like we're ripping off Obsidian's identity.

**Reply by root-hal9000** [reply]
Oh, I didn't think about obsidian. I am not particular to the purple either, but colors aside, I like this streamlined design over the original number 3. But do you think we need to pick different colors from the original Trilium?

**Reply by Jerakin** [reply]
Still thinking about this 

Made a "kind of true to color and shape" version
![image](https://github.com/user-attachments/assets/0164bcd3-fdc1-4da4-8436-cda97debb34f)


edit: versions
![image](https://github.com/user-attachments/assets/9cdcbeee-ca8c-41ce-8f1f-076d8f8f454a)

**Comment by AlexanderTheGrey** [comment]
Out of everything posted here, I prefer 5A, which is essentially a lighter colored version of the original Trilium logo rotated 45 degrees to the left. I guess the point of making the logo purple is to compete with Obsidian? I don't think it's very fitting for this type of logo, but it could catch someone's eye if they use Obsidian.

The original Trilium logo is good too.

**Reply by Alumniminium** [reply]
I had no idea about obsidian or their logo, but that's a fair point i guess.

**Reply by Jerakin** [reply]
I also didn't know obsidian existed. However, I have to admit that I am not a fan of purple logos in general.

As the original is released under AGPL we could reuse the old icon without any (legal) implication afaik. However, as the TriliumNext might end up differ a lot from the source it would be a good idea to change the branding, at least slightly, to differentiate between this project and the upstream one. Setting aside the legal side, I do also think that if the original creator feels strongly about the brand he created for Trilium, and want to keep it on the upstream project then that should carry a lot of weight and be a big reason for this project to change it.

**Reply by Alumniminium** [reply]
I don't get this "zadam cares for the brand" stuff that everyone keeps bringing up when he literally said the following:

> I don't have any copyright or trademark on the Trilium name, the icon etc, although **I think it would be practical** to differentiate the new fork somehow.

But I do agree with your point. We must differentiate properly so people know what version they use. The only way in my opinion is to change the logo in a meaningful way, not just rotate it or change the gradients. Icons are small, they will look the same to anyone not majorly invested into trilium/next

For example if you showed me any of the logos above except for the single color ones in red, orange or purple and asked me what logo that was, I'd say Trilium. For all of them.

**Reply by Jerakin** [reply]
The reason I even made a version of the logo is because I didn't think that the others (including the upstream one) didn't really look like a trillium flower.

I do think that just because the project is named after the flower (well, at least I think it is) the logo doesn't have to look like it. It can be something completely different after all Nike's logo isn't an image of a god and Amazon's isn't an image of a forest (or female warrior). It is of course a lot more common that the logo is tied to the name, but isn't set in stone. I think you could keep the Trilium name and have the logo be something different if someone got an idea (or maybe keep it a flower but either make it more abstract or less abstract).

**Comment by Nriver** [comment]
Am I the only one who misses the old logo?  : D

![256x256](https://github.com/user-attachments/assets/81d7aaeb-5307-4846-a486-bc196381d5ea)

**Reply by zerebos** [reply]
Probably 😄

**Reply by Jerakin** [reply]
That one kind of looks like a trillium flower at least 😃

**Reply by meichthys** [reply]
I miss it!
Simple and true to the flower.

**Comment by meichthys** [comment]
Could I suggest something significantly more simple?
This is just an idea, but we could modify the colors or angles to make it more uniform.

![originalFilename2024-07-19_08-26-34_1217](https://github.com/user-attachments/assets/2890d15e-af2b-4300-8dab-b2145fa1f379)

**Reply by Alumniminium** [reply]
my new favorite

**Comment by eliandoran** [comment]
Since additional candidates have been contributed, voting has been moved to https://github.com/orgs/TriliumNext/discussions/261.
